h2o-genmodel/src/test/java/hex/genmodel/MojoPipelineBuilderTest.java

```diff
@@ -0,0 +1,95 @@
+package hex.genmodel;
+
+import com.google.common.io.ByteStreams;
+import hex.genmodel.easy.EasyPredictModelWrapper;
+import hex.genmodel.easy.RowData;
+import hex.genmodel.easy.exception.PredictException;
+import hex.genmodel.easy.prediction.RegressionModelPrediction;
+import hex.genmodel.tools.BuildPipeline;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import java.io.*;
+
+import static org.junit.Assert.*;
+
+public class MojoPipelineBuilderTest {
+
+  @Rule
+  public TemporaryFolder tmp = new TemporaryFolder();
+
+  private MojoPipelineBuilder builder;
+  private File targetMojoPipelineFile;
+  private File kmeansMojoFile;
+  private File glmMojoFile;
+
+  @Before
+  public void setup() throws IOException {
+    builder = new MojoPipelineBuilder();
+    targetMojoPipelineFile = tmp.newFile("mojo-pipeline.zip");
+    kmeansMojoFile = copyMojoFileResource("kmeans_model.zip");
+    glmMojoFile = copyMojoFileResource("glm_model.zip");
+  }
+
+  @Test
+  public void testPipelineBuilder() throws IOException, PredictException {
+    builder
+            .addModel("clustering", kmeansMojoFile)
+            .addMapping("CLUSTER", "clustering", 0)
```

Comment:
Why is this one upper case ?

Reply:
This is because the column names in MOJO are all uppercase, user needs to match the column names exactly.